### PR TITLE
Updating Treafik Ingress Controller

### PR DESCRIPTION
**Pull request by v-fearam**
Updating treafik version (https://doc.traefik.io/traefik/v2.4/migration/v2/#ingressclass)
Updating Managed Cluster API Version

### CODE REVIEW

File: inner-loop-scripts/shell/1-cluster-stamp.sh [hunk 1]
```diff
@@ -74,7 +74,7 @@ kubectl wait --namespace cluster-baseline-settings --for=condition=ready pod --s
 
 ACR_NAME=$(az deployment group show -g $RGNAMECLUSTER -n cluster-0001 --query properties.outputs.containerRegistryName.value -o tsv)
 # Import ingress controller image hosted in public container registries
-az acr import --source docker.io/library/traefik:2.2.1 -n $ACR_NAME
+az acr import --source docker.io/library/traefik:v2.4.8 -n $ACR_NAME
```
Comment (ckittel) [inline inner-loop-scripts/shell/1-cluster-stamp.sh:77]: YOu need to do this in `08-secret-managment-and-ingress-controller.md` as well, right?

**Review by ckittel — CHANGES_REQUESTED** [review]
I didn't test end to end, assume you did.  Only one obersvation that will prevent folks from completing this if they are using the readme files.

File: inner-loop-scripts/shell/1-cluster-stamp.sh (hunk 1 above)
Comment (v-fearam) [reply inner-loop-scripts/shell/1-cluster-stamp.sh:77]: Yes. Good catch. Thanks. Fixed

**Comment by v-fearam** [comment]
yes. I tested end to end before creating the pull request.

File: inner-loop-scripts/shell/1-cluster-stamp.sh (hunk 1 above)
Comment (ckittel) [reply inner-loop-scripts/shell/1-cluster-stamp.sh:77]: coo!

**Review by ckittel — APPROVED** [review]
🚢

**Pull request merged**